### PR TITLE
Rename properties related to Stickers

### PR DESCRIPTION
**Pull request by andybalaam**
Small refactoring, should not change behaviour at all.

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

Preview: https://61f3cba0e04f07a44b9b5e99--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/views/rooms/MessageComposer.tsx [hunk 1]
```diff
@@ -255,7 +255,7 @@ interface IState {
     me?: RoomMember;
     narrowMode?: boolean;
     isMenuOpen: boolean;
-    showStickers: boolean;
+    stickersVisible: boolean;
```
Comment (t3chguy) [inline src/components/views/rooms/MessageComposer.tsx:258]: maybe `isStickerPickerOpen` to match `isMenuOpen` above

File: src/components/views/rooms/Stickerpicker.tsx [hunk 2]
```diff
@@ -155,16 +155,16 @@ export default class Stickerpicker extends React.PureComponent<IProps, IState> {
         }
     }
 
-    public componentDidUpdate(prevProps: IProps, prevState: IState): void {
-        this.sendVisibilityToWidget(this.props.showStickers);
+    public componentDidUpdate(_prevProps: IProps, _prevState: IState): void {
```
Comment (t3chguy) [inline src/components/views/rooms/Stickerpicker.tsx:158]:
```suggestion
    public componentDidUpdate(): void {
```

File: src/components/views/rooms/Stickerpicker.tsx [hunk 3]
```diff
@@ -123,7 +123,7 @@ export default class Stickerpicker extends React.PureComponent<IProps, IState> {
             logger.warn('No widget ID specified, not disabling assets');
         }
 
-        this.props.setShowStickers(false);
+        this.props.setStickersVisible(false);
```
Comment (t3chguy) [inline src/components/views/rooms/Stickerpicker.tsx:126]: in the same vein maybe `setStickerPickerOpen(...)`

File: src/components/views/rooms/MessageComposer.tsx (hunk 1 above)
Comment (andybalaam) [reply src/components/views/rooms/MessageComposer.tsx:258]: I like it, thanks.

Comment (andybalaam) [reply src/components/views/rooms/MessageComposer.tsx]: Done in 0e9fd3e49f

File: src/components/views/rooms/Stickerpicker.tsx (hunk 3 above)
Comment (andybalaam) [reply src/components/views/rooms/Stickerpicker.tsx]: Done in 0e9fd3e49f

File: src/components/views/rooms/Stickerpicker.tsx [hunk 4]
```diff
@@ -113,7 +113,7 @@ export default class Stickerpicker extends React.PureComponent<IProps, IState> {
             if (scalarClient) {
                 scalarClient.disableWidgetAssets(WidgetType.STICKERPICKER, this.state.widgetId).then(() => {
                     logger.log('Assets disabled');
-                }).catch((err) => {
+                }).catch((_err) => {
```
Comment (t3chguy) [inline src/components/views/rooms/Stickerpicker.tsx:116]:
```suggestion
                }).catch(() => {
```

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

**Pull request merged**